vendor/src/github.com/docker/libnetwork/sandbox/namespace_linux.go

```diff
@@ -26,11 +37,52 @@ type networkNamespace struct {
 	sync.Mutex
 }
 
+func init() {
+	reexec.Register("netns-create", reexecCreateNamespace)
+}
+
 func createBasePath() {
 	err := os.MkdirAll(prefix, 0644)
 	if err != nil && !os.IsExist(err) {
 		panic("Could not create net namespace path directory")
 	}
+
+	// cleanup any stale namespace files if any
+	cleanupNamespaceFiles()
```

Comment:
I'm not sure it's a supported use case, but I'm pretty sure reading some people start multiple daemons, and in this case this call might be very problematic.